config/core/resources/configuration.yaml

```diff
@@ -22,16 +22,23 @@ metadata:
     duck.knative.dev/podspecable: "true"
 spec:
   group: serving.knative.dev
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      # this is a work around so we don't need to flush out the
+      # schema for each version at this time
```

Comment:
Can you add a link to https://github.com/knative/serving/issues/912 ?

Reply:
done